lib/Driver/ToolChains.cpp

```diff
@@ -1182,13 +1212,9 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
         Twine("@") + OutputInfo.getPrimaryOutputFilename()));
   }
 
-  // Add the linker script that coalesces protocol conformance sections.
-  Arguments.push_back("-Xlinker");
-  Arguments.push_back("-T");
-    
-  // FIXME: This should also query the abi type (i.e. gnueabihf)
   Arguments.push_back(context.Args.MakeArgString(
-    Twine(RuntimeLibPath) + "/" + getTriple().getArchName() + "/swift.ld"));
+    Twine(RuntimeLibPath) + "/" + getTriple().getArchName() +
```

Comment:
It's important to note that we have to be sure this is after all the objects and libraries being linked, if a static swift library is part of the link, this object must follow them, that's why we decided to place it at the very end.